src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1729,20 +1731,31 @@ public static bool TryConvertTo(object valueToConvert, Type resultType, out obje
         /// <returns>false for conversion failure, true for success</returns>
         public static bool TryConvertTo(object valueToConvert, Type resultType, IFormatProvider formatProvider, out object result)
         {
-            result = null;
+            result = default;
```

Comment:
I think `result = null` is more explicit here, as the type of `result` is `object` here.